docs/docs/Get-Started/get-started-installation.md

```diff
@@ -6,179 +6,205 @@ slug: /get-started-installation
 
-```bash
-pip install langflow -U
-```
+To install and run Langflow OSS, you need the following:
 
-</TabItem>
-</Tabs>
+- [Python 3.10 to 3.13](https://www.python.org/downloads/release/python-3100/) for macOS/Linux, and Python 3.10 to 3.12 for Windows
+- [uv](https://docs.astral.sh/uv/getting-started/installation/)
+- At minimum, a dual-core CPU and 2 GB RAM, but a multi-core CPU and at least 4 GB RAM are recommended
 
-To install a specific version of the Langflow package, add the required version to the command.
-<Tabs groupId="package-manager">
-<TabItem value="uv" label="uv" default>
+1. Create a virtual environment with [uv](https://docs.astral.sh/uv/pip/environments).
 
-```bash
-uv pip install langflow==1.3.2
-```
+<details>
+<summary>Need help with virtual environments?</summary>
 
-</TabItem>
-<TabItem value="pip" label="pip">
+Virtual environments ensure Langflow is installed in an isolated, fresh environment.
+To create a new virtual environment, do the following.
 
-```bash
-pip install langflow==1.3.2
+<Tabs groupId="os">
+  <TabItem value="macOS/Linux" label="macOS/Linux" default>
+    1. Navigate to where you want your virtual environment to be created, and create it with `uv`.
+Replace `VENV_NAME` with your preferred name for your virtual environment.
+```
+uv venv VENV_NAME
+```
+2. Start the virtual environment.
+```
+source VENV_NAME/bin/activate
 ```
+Your shell's prompt changes to display that you're currently working in a virtual environment.
+```
+(VENV_NAME) ➜  langflow git:(main) ✗
+```
+3. To deactivate the virtual environment and return to your regular shell, type `deactivate`.
+To delete the virtual environment, type `rm -rf VENV_NAME`.
+  </TabItem>
+  <TabItem value="Windows" label="Windows">
+1. Navigate to where you want your virtual environment to be created, and create it with `uv`.
+Replace `VENV_NAME` with your preferred name for your virtual environment.
+```
+uv venv VENV_NAME
+```
+2. Start the virtual environment.
+```shell
+VENV_NAME\Scripts\activate
+```
+Your shell's prompt changes to display that you're currently working in a virtual environment.
+```
+(VENV_NAME) PS C:/users/username/langflow-dir>
+```
+3. To deactivate the virtual environment and return to your regular shell, type `deactivate`.
+To delete the virtual environment, type `Remove-Item VENV_NAME`.
+  </TabItem>
+  </Tabs>
 
-</TabItem>
-</Tabs>
+</details>
 
-To reinstall Langflow and all of its dependencies, add the `--force-reinstall` flag to the command.
-<Tabs groupId="package-manager">
-<TabItem value="uv" label="uv" default>
+2. To install Langflow, run the following command.
+  ```bash
+  uv pip install langflow
+  ```
 
-```bash
-uv pip install langflow --force-reinstall
-```
+3. After installation, start Langflow:
+  ```bash
+  uv run langflow run
+  ```
 
-</TabItem>
-<TabItem value="pip" label="pip">
+4. To confirm that a local Langflow instance is running, navigate to the default Langflow URL`http://127.0.0.1:7860`.
```

Comment:
```suggestion
4. To confirm that a local Langflow instance is running, navigate to the default Langflow URL `http://127.0.0.1:7860`.
It can take a few minutes for Langflow to start.
```